### PR TITLE
chore(rxFeedback): decompose monolithic source

### PR DESCRIPTION
**Pull request by CITguy**
[FRMW-350](https://jira.rax.io/browse/FRMW-350)

### LGTMS
- [x] Dev LGTM
- [x] Dev LGTM

### CODE REVIEW

File: src/rxFeedback/rxFeedback.js [hunk 1]
```diff
@@ -1,353 +0,0 @@
...
- *
- * ## Directives
- * * {@link rxFeedback.directive:rxFeedback rxFeedback}
- *
- * ## Services
- * * {@link rxFeedback.service:rxFeedbackSvc rxFeedbackSvc}
- * * {@link rxFeedback.service:rxScreenshotSvc rxScreenshotSvc}
- */
-angular.module('encore.ui.rxFeedback', ['ngResource'])
-.value('feedbackTypes', [
```
Comment (CITguy) [inline src/rxFeedback/rxFeedback.js]: moved to `src/rxFeedback/scripts/feedbackTypes.js`

File: src/rxFeedback/rxFeedback.js [hunk 2]
```diff
@@ -1,353 +0,0 @@
...
-])
-/**
- * @ngdoc service
- * @name rxFeedback.service:rxScreenshotSvc
- * @description
- * Captures a screenshot for `rxFeedback` submission form.
- *
- * **NOTE:** Requires `html2canvas` which Encore Framework provides by default.
- */
-.service('rxScreenshotSvc', function ($log, $q) {
```
Comment (CITguy) [inline src/rxFeedback/rxFeedback.js]: moved to `src/rxFeedback/scripts/rxScreenshotSvc.js`

File: src/rxFeedback/rxFeedback.js [hunk 3]
```diff
@@ -1,353 +0,0 @@
...
-    };
-})
-/**
- * @ngdoc service
- * @name rxFeedback.service:rxFeedbackSvc
- * @description
- * `rxFeedbackSvc` service supports `rxFeedback` directive functionality.  A `custom endpoint` may be set to override
- * the `default` endpoint.
- */
-.factory('rxFeedbackSvc', function ($resource, feedbackApi, $location, $window) {
```
Comment (CITguy) [inline src/rxFeedback/rxFeedback.js]: moved to `src/rxFeedback/scripts/rxFeedbackSvc.js`

File: src/rxFeedback/rxFeedback.js [hunk 4]
```diff
@@ -1,353 +0,0 @@
...
-    return container;
-})
-/**
- * @ngdoc controller
- * @name rxFeedback.controller:rxFeedbackController
- * @scope
- * @description
- * Allows the customization of the feedback modal via `$scope` and `$modalInstance`.
- */
-.controller('rxFeedbackController', function ($scope, $modalInstance, $rootScope, $injector) {
```
Comment (CITguy) [inline src/rxFeedback/rxFeedback.js]: moved to `src/rxFeedback/scripts/rxFeedbackController.js`

File: src/rxFeedback/rxFeedback.js [hunk 5]
```diff
@@ -1,353 +0,0 @@
...
- *    },
- *    "description": "(string) user-submitted feedback"
- * }
- * </pre>
- *
- * @param {Object} type JSON object with `label` {String}, `placeholder` {String}, and `prompt` {String}
- * @param {String} description User-submitted feedback
- *
- */
-.directive('rxFeedback', function (feedbackTypes, $location, rxFeedbackSvc, rxScreenshotSvc, rxNotify, Session) {
```
Comment (CITguy) [inline src/rxFeedback/rxFeedback.js]: moved to `src/rxFeedback/scripts/rxFeedback.js`

File: src/rxFeedback/rxFeedback.spec.js [hunk 6]
```diff
@@ -1,432 +0,0 @@
-/* jshint node: true */
-describe('rxFeedback', function () {
```
Comment (CITguy) [inline src/rxFeedback/rxFeedback.spec.js]: moved to `src/rxFeedback/scripts/rxFeedback.spec.js`

File: src/rxFeedback/rxFeedback.spec.js [hunk 7]
```diff
@@ -1,432 +0,0 @@
...
-        it('should apply the custom feedback function for on-submit', function () {
-            elCustomScope.sendFeedback();
-            expect(scope.kitchenSink).to.be.calledOnce;
-        });
-
-    });
-
-});
-
-describe('rxScreenshotSvc', function () {
```
Comment (CITguy) [inline src/rxFeedback/rxFeedback.spec.js]: moved to `src/rxFeedback/scripts/rxScreenshotSvc.spec.js`

File: src/rxFeedback/rxFeedback.spec.js [hunk 8]
```diff
@@ -1,432 +0,0 @@
...
-            expect(screenshot).to.exist;
-
-            done();
-        });
-
-        scope.$digest();
-    });
-});
-
-describe('rxFeedbackSvc', function () {
```
Comment (CITguy) [inline src/rxFeedback/rxFeedback.spec.js]: moved to `src/rxFeedback/scripts/rxFeedbackSvc.spec.js`

File: src/rxFeedback/rxFeedback.spec.js [hunk 9]
```diff
@@ -1,432 +0,0 @@
...
-        feedbackSvc.email = ninetiesEmail;
-
-        mockWindow.open.returns(undefined);
-        feedbackSvc.fallback(feedback);
-
-        expect(mockWindow.location.href).to.contain(ninetiesEmail);
-    });
-});
-
-describe('rxFeedbackController', function () {
```
Comment (CITguy) [inline src/rxFeedback/rxFeedback.spec.js]: moved to `src/rxFeedback/scripts/rxFeedbackController.spec.js`

**Comment by bry4n** [comment]
LGTM

**Comment by 100stacks** [comment]
LGTM

**Pull request merged**